.github/workflows/test.yml

```diff
@@ -71,3 +71,99 @@ jobs:
+      run:
+        working-directory: Meadowlark-js
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+
+      - name: NPM caching
+        uses: actions/setup-node@17f8bd926464a1afa4c6a11669539e9c1ba77048 # v3.2.0
+        with:
+          node-version: "16"
+          cache: "yarn"
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Start OpenSearch Container
+        run: docker compose -f ./backends/meadowlark-opensearch-backend/docker/docker-compose.yml up -d
+
+      - if: ${{ matrix.config.db == 'PGSQL' }}
+        name: Start Postgres Container
+        run: docker compose -f ./backends/meadowlark-postgresql-backend/docker/docker-compose.yml up -d
+
+      - if: ${{ matrix.config.db == 'Mongo' }}
+        name: Start MongoDB
+        uses: supercharge/mongodb-github-action@538a4d2a1041920c47630172445cb688592d6e51 #v1.8.0
+        with:
+          mongodb-version: "4.0.28"
+          mongodb-replica-set: rs0
+          mongodb-port: 42069
+
+      - name: Cache Build
+        uses: actions/cache@48af2dc4a9e8278b89d7fa154b955c30c6aaab09 #v3.0.2
+        env:
+          cache-name: cache-build
+        with:
+          path: "**/dist/**"
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/dist/**') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+
+      - if: ${{ steps.cache-build.outputs.cache-hit != 'true' }}
+        name: Build
+        run: yarn build
+
+      - name: Run Fastify in background
+        run: |
+          cp ./tests/e2e/setup/e2e.env ./services/meadowlark-fastify/.env
+          yarn start:local > fastify.log &
+        env:
+          DOCUMENT_STORE_PLUGIN: ${{ matrix.config.plugin }}
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: abcdefgh1!
+          OPENSEARCH_USERNAME: admin
+          OPENSEARCH_PASSWORD: admin
+          MONGO_URL: "mongodb://127.0.0.1:42069/?replicaSet=rs0"
+
+      - name: End to End tests for ${{matrix.config.db}}
+        run: |
+          cp ./tests/e2e/.env.example ./tests/e2e/.env
+          yarn test:e2e --ci --config ./jest.ci-config.js
+        env:
+          JEST_JUNIT_OUTPUT_DIR: ${{matrix.config.db}}-e2e-tests
+
+      - name: Upload logs
+        if: failure()
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+        with:
+          name: fastify-logs
+          path: "**/fastify.log"
+          retention-days: 10
+
+      # Workaround for issue https://github.com/actions/upload-artifact/issues/174
+      - name: Upload test results
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+        with:
+          name: test-results
+          path: |
+            Meadowlark-js/${{matrix.config.db}}-e2e-tests/junit.xml
+            Meadowlark-js/.prettierrc
```

Comment:
The reporter issue causes that when a single file is attached, the folder structure is not preserved, to maintain the structure, it's necessary to attach any other file into the generated artifact.